<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateHallDialog</class>
 <widget class="QMainWindow" name="CreateHallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background: #22272E;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget { background: #22272E;}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>791</width>
      <height>591</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>791</width>
      <height>591</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {border: none;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: #ADBAC7;}</string>
     </property>
     <property name="text">
      <string>Prices</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources.qrc">:/assets/cute_pig_bun35.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>520</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
       border: 2px solid #ADBAC7; border-radius: 20px; background: #22272E;
       color: #ADBAC7; font-size: 20px; padding: 10px; padding-left: 20px;
       padding-right: 20px;
} 

QPushButton:hover {
		border-color: #8598ab; 
		color: #8598ab;
}</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>No. of baby pigs</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>248</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Start with Little boxes ?</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>248</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Custom size ?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lenght">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Lenght</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>No. of adult pigs</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_total_price">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>440</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>TOTAL PRICE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {color: #ADBAC7; font-size: 20px;}</string>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
